src/Entities/Entity.h

```diff
@@ -42,6 +42,7 @@
 		} \
 	} while (false)
 
+#define VOLUME GetWidth() * GetHeight()
```

Comment:
We don't normally use macros like this anymore, the ones above are legacy. If this is used in loads of places it could be added as a private member function, but I don't think this needs abstraction at all honestly, just copy the calculation whenever it's used.